src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -2109,8 +2117,16 @@ public void setSelectAllFiles(boolean selectAllFiles) {
         this.selectAllFiles = selectAllFiles;
     }
     
-    public void toggleSelectedFiles(){
-        //method for when user clicks (de-)select all files
+    public void toggleAllSelected(){
+        System.out.print("in toggle all");
```

Comment:
Can you please remove this `System.out.print` or convert it to `logger.fine` if it might be valuable when debugging in production?

Reply:
Done